e2e/release/release_test.go

```diff
@@ -47,42 +47,44 @@ func TestRelease(t *testing.T) {
 	ctx := context.Background()
 	k := kubeclient.NewForTest(t)
 
-	if *namespace == "" {
-		*namespace = randomNamespace(t)
```

Comment:
If we test in the default namespace, we don't need the `randomNamespace` function anymore.